### DISCUSSION TITLE
CSP

### DISCUSSION THREAD

**Discussion by harrisonobaz** [cake-build/cake#3402] [Extension Q&A]
How to build a CSP project using MongoDB.Driver.Core.2.12.4

**Comment by pascalberger** [comment]
I'm not aware what a CSP project is or any MongoDB specific addins. But Cake is just a build orchestration tool, which calls other tools to do the actual work. If you now how to build your project on the command line, you always can just make the same call using any of the [Process aliases](https://cakebuild.net/dsl/process/#General).

**Comment by harrisonobaz** [comment]
Apologies. It’s a communication service provider project am working on, I’m looking to gain a better idea on how to build it on the command line, and add-ins I’ll be needing for the project.
